README.md

```diff
@@ -44,6 +44,17 @@ A collaborative effort to build the fattest Mach-O binary in existence.
 * **Built on:** 07/22/2020 using Xcode Version 3.1.4 (9M2809) on a MacBook Pro (Retina, 13-inch, Early 2015) running macOS Mojave 10.14.6 (18G5033)
 * **Physically tested by:** [@foxlet](https://github.com/foxlet) on 07/22/2020, using a PowerBook G4 (1 GHz) running Mac OS X Leopard 10.5.8 (9L31a)
 
+### [Motorola 680x0 (m68k)](https://github.com/saagarjha/dummy_thicc/tree/master/m68k)
+
+* **Built on:** 2020-08-31 using GCC 2.95.3 on an emulated Next Computer running NeXTSTEP 3.3 (running in Previous 1.8)
+* **Virtually tested by**: [@homsar](https://github.com/homsar) on 2020-08-31, emulating a Next Computer running NeXTSTEP 3.3
```

Comment:
Nit: [Wikipedia calls it the "NeXT Computer"](https://en.wikipedia.org/wiki/NeXT_Computer) so I'm going to lean on that being its official name. The others are just to make my OCD feel better, and to use that stupid American date system :P

```suggestion
* **Built on:** 08/31/2020 using GCC 2.95.3 on an emulated NeXT Computer running NeXTSTEP 3.3 in Previous 1.8
* **Virtually tested by**: [@homsar](https://github.com/homsar) on 08/31/2020 emulating a NeXT Computer running NeXTSTEP 3.3 in Previous 1.8
```

Reply:
I was copying the format used by Previous, which is absolutely not guaranteed to be right, so happy to go with Wikipedia here. Keeping "NeXT" consistent makes me happy.

Reply:
I use ISO 8601 everywhere (even where it gets me odd looks), but like I said below, your repository, your rules. (If you asked me to [carve the numbers onto the side of a cat](https://xkcd.com/1179/) then I would however withdraw the PR.)